pkg/assets/static/index.html

```diff
@@ -5,6 +5,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <link rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly@4.6.3/patternfly.css" crossorigin="anonymous">
   <link rel="stylesheet" href="landingpage.css">
+  <script src="https://www.google.com/recaptcha/enterprise.js?render=6LcaESslAAAAAO1aPnbNiP1fg6z9VC5Y33Rq-Sl4"></script>
```

Comment:
I'm curious what is that `6LcaESslAAAAAO1aPnbNiP1fg6z9VC5Y33Rq-Sl4`? is it some kind of our account/client ID or something?

Reply:
It's the recaptcha site-key ID. See https://cloud.google.com/recaptcha-enterprise/docs/create-key There will be different site keys for stage and prod but this is one we can use for dev/test. In our ToolchainConfig configuration it's the `Captcha.SiteKey`.